src/charts/bar.js

```diff
@@ -317,6 +328,81 @@ define(function(require) {
         }
 
         /**
+         * Draws and animates the bars along the x axis
+         * @param  {D3Selection} bars Selection of bars
+         * @return {void}
+         */
+        function drawAnimatedHorizontalBars(bars) {
+            // Enter + Update
```

Comment:
Yeah, totally. I am not super happy with this approach either.
I will think about how to do it other way...